### ISSUE TITLE
Legend labeling simplification

### ISSUE THREAD

**Issue by smit1678** [hotosm/field-campaigner#380]
Number and labeling of categories is slightly hard to parse visually. 

![map campaigner - boulder cafes 2017-10-30 10-45-45](https://user-images.githubusercontent.com/796838/32166998-7f27812c-bd5f-11e7-9b1d-2aa974cfe3d7.png)

Idea for adjustment: 

100% - green
75% - yellow
50% - dark yellow
25% - orange
0% - red

@ann26 thoughts?

**Comment by PaulUithol** [comment]
Yeah, sounds good - still have a draft issue around (in a tab) with other areas we'd want to add/amend text

**Comment by ann26** [comment]
Something like this @smit1678 ?
![screenshot from 2017-10-31 09-43-48](https://user-images.githubusercontent.com/26101337/32205131-16d79884-be20-11e7-8ad0-439d08f0d600.png)

with the value completeness:
100% means 100% complete
75% means 75% - <100% complete
50% means 50% - <75% complete
25% means 25% - <50% complete
0% means <25% complete

**Comment by smit1678** [comment]
@ann26 👍  yes, this looks good
